### PR TITLE
fix for redirection error of embedded search

### PR DESCRIPTION
**Pull request by nooraangelva**
Ref: #1081

### CODE REVIEW

File: ui/src/actions/search.js [hunk 1]
```diff
@@ -34,10 +34,11 @@ function searchSuccess(namespace, data) {
 }
 
 function searchError(namespace, errorPayload) {
+  const { redirectableError } = searchConfig[namespace].redirectableError;
```
Comment (drjova) [inline ui/src/actions/search.js:37]:
```suggestion
  const { redirectableError } = searchConfig[namespace];
```

File: ui/src/actions/__tests__/search.test.js [hunk 2]
```diff
@@ -42,8 +42,8 @@ describe('search - action creators', () => {
       const namespace = AUTHOR_PUBLICATIONS_NS;
       const pathname = LITERATURE;
       const store = getStore();
-      const url = `${pathname}?page=1&size=10&q=test`;
-      mockHttp.onGet(`${pathname}?page=1&size=10&q=test`).networkError();
+      const url = `${pathname}/1787272`;
+      mockHttp.onGet(`${pathname}/1787272`).networkError();
```
Comment (drjova) [inline ui/src/actions/__tests__/search.test.js:46]: why do you change those?

File: ui/src/search/config.js [hunk 3]
```diff
@@ -39,74 +39,92 @@ const searchConfig = {
...
   },
   [EXPERIMENTS_NS]: {
     persistedQueryParamsDuringNewSearch: defaultPersistedQueryParamsDuringNewSearch,
     onQueryChange: onCollectionQueryChange,
+    redirectableError: true,
   },
   [EXPERIMENT_PAPERS_NS]: {
     persistedQueryParamsDuringNewSearch: persistedQueryParamsDuringNewSearchForEvents,
     onQueryChange: onEmbeddedLiteratureQueryChange,
+    edirectableError: false,
```
Comment (drjova) [inline ui/src/search/config.js:112]:
```suggestion
    redirectableError: false,
```

File: ui/src/actions/__tests__/search.test.js [hunk 4]
```diff
@@ -52,34 +52,54 @@ describe('search - action creators', () => {
         {
           type: types.SEARCH_ERROR,
           payload: { namespace, error: { status: 'network' } },
-          meta: { redirectableError: true }, // TODO: should this be redirectableError?
+          meta: { redirectableError: true },
```
Comment (drjova) [inline ui/src/actions/__tests__/search.test.js:55]: @nooraangelva why did you change this test? we should add a new one not modify the existing ones.

Comment (nooraangelva) [reply ui/src/actions/__tests__/search.test.js:55]: I did add a new one and this is the new one. 
The other test (old one) is [here](https://github.com/inspirehep/inspirehep/blob/fc65ac2313b1cc6c17354ac38891c9db565cff3b/ui/src/actions/__tests__/search.test.js#L61-L79) I did have to change the `redirectableError` to `false` since in config it is false.

**Review by drjova — APPROVED** [review]
